rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -1839,33 +1840,41 @@ int LuaSyncedCtrl::DestroyUnit(lua_State* L)
 int LuaSyncedCtrl::TransferUnit(lua_State* L)
 {
 	CheckAllowGameChanges(L);
+	const int args = lua_gettop(L);
```

Comment:
I don't understand this change, how is the 'captured' use case handled now? See also the documentation a few lines above.